### PR TITLE
[kube-prometheus-stack] - prometheus-operator0.59.2/dependencies

### PR DESCRIPTION
**Pull request by Allex1**
Updates Prometheus Operator to 0.59.2, prometheus-node-exporter, kube-state-metrics and Grafana to latest

### CODE REVIEW

**Comment by Allex1** [comment]
@Xtigyro @monotek @mrueg can you pls take a look ?

**Review by dotdc — DISMISSED** [review]
LGTM

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -22,8 +22,8 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 40.2.0
-appVersion: 0.59.1
+version: 40.2.1
```
Comment (dotdc) [inline charts/kube-prometheus-stack/Chart.yaml:25]: this should be a minor

Comment (Allex1) [reply charts/kube-prometheus-stack/Chart.yaml:25]: @dotdc do you mean major? as in 41.0.0? no crds are changing.

Comment (dotdc) [reply charts/kube-prometheus-stack/Chart.yaml:25]: Minor, the middle number

Comment (Allex1) [reply charts/kube-prometheus-stack/Chart.yaml]: @dotdc done

**Review by monotek — APPROVED** [review]

**Pull request merged**